### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by Rockyraven**
some add more component and component button

### CODE REVIEW

File: component/avatar/avatar.html [hunk 1]
```diff
@@ -41,6 +41,8 @@ <h1 class="logo">BOOSTWEB UI</h1>
             <a href="/component/navbar/navbar.html">Navbar</a>
             <a href="/component/rating/rating.html">Rating</a>
             <a href="/component/utility/utility.html">Text Utilities</a>
+            <a href="/component/model/model.html">Model</a>
```
Comment (bharati-21) [inline component/avatar/avatar.html]: Please do check your modal spelling.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged. Only check your modal spelling. 👍🏽

File: component/avatar/avatar.html (hunk 1 above)
Comment (Rockyraven) [reply component/avatar/avatar.html]: sorry my mistake I will correct it

**Review by vishalg8454 — APPROVED** [review]
All good to merge.

**Review by Shubhamkr2610 — COMMENTED** [review]
Looks good to me

**Pull request merged**